Source/WebCore/style/values/svg/StyleSVGPaint.cpp

```diff
@@ -27,152 +27,103 @@
 
 #include "AnimationUtilities.h"
 #include "CSSURLValue.h"
-#include "ColorBlending.h"
-#include "RenderStyle.h"
 #include "StyleBuilderChecking.h"
-#include "StyleColor.h"
 #include "StyleForVisitedLink.h"
-#include <wtf/text/TextStream.h>
 
 namespace WebCore {
 namespace Style {
 
-auto CSSValueConversion<SVGPaint>::operator()(BuilderState& state, const CSSValue& value, ForVisitedLink forVisitedLink) -> SVGPaint
+bool containsCurrentColor(const Style::SVGPaint& paint)
 {
-    // <paint> = none | <color> | <url> [none | <color>]? 
+    if (auto color = paint.tryAnyColor())
+        return color->containsCurrentColor();
+    return false;
```

Comment:
Darn, I should have suggested this too:
```suggestion
    auto color = paint.tryAnyColor();
    return color && color->containsCurrentColor();
```